rke/structure_rke_cluster.go

```diff
@@ -505,3 +511,13 @@ func expandRKEClusterFlag(in *schema.ResourceData, clusterFilePath string) clust
 
 	return obj
 }
+
+func k8sVersionRequiresCri(kubernetesVersion string) bool {
+
+	version, err := getClusterVersion(kubernetesVersion)
+	if err != nil {
+		// Ignoring the error, if the k8sVersion is not valid it should fail during the validation of the field.
+		log.Debug("invalid kubernetes version")
```

Comment:
Nit:
- Can we include the k8s version in the msg to make it clear?  
- Maybe we should return false directly here to make logic clear instead of invoking the "return" line even though we know it will always return false in this case?

Reply:
Done. 
I didn't handle the error nor gave much attention  to it cause it is not supposed to happen.